File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/ThanhVienTokenService.cs
namespace Service.Implement
{
    public class ThanhVienTokenService : BaseService<ThanhVienToken, IThanhVienTokenRepository>
    , IThanhVienTokenService
    {
    private readonly IThanhVienTokenRepository _ThanhVienTokenRepository;
    public ThanhVienTokenService(IThanhVienTokenRepository ThanhVienTokenRepository) : base(ThanhVienTokenRepository)
    {
    _ThanhVienTokenRepository = ThanhVienTokenRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/ThanhVienToken.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThanhVienToken } from 'src/app/shared/ThanhVienToken.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThanhVienTokenService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThanhVienToken";
    }
}

